Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedOperation.swift

```diff
@@ -94,8 +94,8 @@ struct UnifiedOperation: Decodable {
                 entities[entityId] = try actualResult.asEntity()
```

Comment:
can't comment on the exact line, but in the next PR we'll start doing a try/catch when executing the operation since some ops expect errors. I'm thinking that for both expected errors (and their assertions) we can pass this context down in, and for unexpected ones we can put it in the message, so it's easier to know which particular operation unexpectedly failed.